CONTRIBUTING.md

```diff
@@ -98,6 +114,58 @@ npm link @wdio/cli
 
 In order to test certain scenarios this project has a test directory that allows you to run predefined test. It allows you to check your code changes while you are working on it. You find all these files in `/examples`. You find all necessary information [in there](https://github.com/webdriverio/webdriverio/tree/master/examples/README.md).
 
+## Back-Porting Bug Fixes
+
+Starting from v6 the WebdriverIO team tries to backport all features that would be still backwards compatible with older versions. The team tries to release a new major version every year (usually around December/Januar). With a new major version update (e.g. v6) we continue to maintain the last version (e.g. v5) and depcrecate the previous maintained version (e.g. v4). With that the team commits to always support 2 major versions.
+
+### As Triager
+
+Everyone triaging or reviewing a PR should label it with `backport-requested` if the changes can be applied to the maintained (previous) version.
+
+### As A Merger
+
+Once a PR with a `backport-requested` label got merged, you are responsible for the backporting the patch to the older version. To do so, pull the latest code from GitHub:
```

Comment:
`you are responsible for the backporting the patch` -> you are responsible for backporting the patch